cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperatorTest.java

```diff
@@ -1001,7 +1001,7 @@ private void updateCluster(VertxTestContext context, Kafka originalAssembly, Kaf
                 .withNewMetadata()
                     .withName("metrics-cm")
                 .endMetadata()
-                .withData(singletonMap("metrics-config.yml", ""))
+                .withData(singletonMap("metrics-config.json", ""))
```

Comment:
Are you sure this should be `.json`? This looks like the user Config Map to me.

Reply:
@scholzj Yes I guess, If we run the tests with `.yml`, then we get a InvalidResourceEception error, since its searching for the key `metrics-config.json` in `metrics-cm'